templates/generate.html

```diff
@@ -799,6 +829,10 @@ <h5 class="card-title">Generated Output</h5>
                 options.num_predict = parseInt(maxTokens.value);
             }
             
+            if (contextSize.value) {
+                options.num_ctx = parseInt(contextSize.value);
+            }
+            
```

Comment:
The addition of the `num_ctx` parameter for adjusting context size is a valuable feature. It's handled well here, with proper conversion to an integer before being passed to the Ollama API. This ensures that the payload is correctly formatted.